cfpb/consumerfinance.gov
# Update the Rental Assistance Finder tool

This updates the site to use version `0.5.3` of `rental-assistance-finder`. 

This will change content that is hard-coded into the external app, but makes no changes to functionality.

## Changes
-  Updates `rental-assistance-finder` to `0.5.3`


## How to test this PR
1. Get it running and check the text on the Rental Assistance Finder page to make sure it's been updated.

## Checklist
